package.json

```diff
@@ -27,6 +27,7 @@
     "leaflet": "^1.7.1",
     "lodash": "^4.17.15",
     "lodash-webpack-plugin": "^0.11.4",
+    "luxon": "^1.27.0",
```

Comment:
good to remove dependency from `package.json`

Reply:
How come? Doesn't it need that dependency to run since those pages are dynamically created?